### ISSUE TITLE
While navigating, allow auto-orienting map toward the destination

### ISSUE THREAD

**Issue by sknorr** [osmandapp/Osmand#7683]
While using Osmand for navigation as a cyclist, I have found that the existing map orientation options have some significant disadvantages:

* "No rotation (use North)"
  * can be really irritating when moving south; at least my brain regularly makes bad decisions where to go left and where to go right when the on-screen arrow and the view of the map collide
  * you can rotate the map, but it will always be reset to be oriented toward the North when locking the phone 
* "By direction"/"By compass"
  * way too much overall map movement, every corner you turn may mean a 40+ degree map rotation

So, what I'd love is an option that, when navigation is running, always orients the map toward my destination.

Not sure whether intermediate destination should be taken into account here or whether it should always be the overall destination. However, to improve the use case of round-trips or similar, it might be better to use intermediate destinations as well.

**Comment by jojoob** [comment]
There are (now?) four map orientation options North up, Compass, Manually and Movement direction. Maybe "Movement direction" fits your need?

**Comment by Famlam** [comment]
Neither of these point towards your destination, right? (Unless you're moving in a straight line towards it)

**Comment by scaidermern** [comment]
>     * "By direction"/"By compass"
>       
>       * way too much overall map movement, every corner you turn may mean a 40+ degree map rotation

Wouldn't the result be the same if the map points towards the destination? If you turn 40 degree, your device will turn 40 degree, therefore your map has to turn 40 degree in order to point into the same direction as before.

**Comment by Famlam** [comment]
Destination will always stay at the top of your screen regardless of device orientation with this request, not?

**Comment by jojoob** [comment]
Yes, that makes sense. Seems Famlam and I misunderstood the request.
